src/components/mandataireComponents/CreationMesure.js

```diff
@@ -44,6 +52,23 @@ export const FormMesure = ({
             error={error}
             status={status}
             success={success}
+            value={value}
+            lieuxDeVie={lieuxDeVie}
+            updateLieuxDeVie={updateLieuxDeVie}
+            updateValue={updateValue}
+            etablissement={etablissement}
+            onChange={onChange}
+            formDataState={formDataState}
+            render={() => (
+              <LieuxDeVie
+                {...this.props}
+                value={value}
+                lieuxDeVie={lieuxDeVie}
+                updateLieuxDeVie={updateLieuxDeVie}
+                updateValue={updateValue}
+                etablissement={etablissement}
+              />
+            )}
```

Comment:
Je ne vois pas l'intérêt d'utiliser une "render prop" ici. Ca me paraitrait utile si `FormInputMesure` exécutait `render()` avec des paramètres mais visiblement, ici, non ?

Ce pourrait marcher pareil en utilisant uniquement `{this.props.children}`  dans `FormInputMesure` ?

```js
<FormInputMesure......>
  <LieuxDeVie
      {...this.props}
      value={value}
      lieuxDeVie={lieuxDeVie}
      updateLieuxDeVie={updateLieuxDeVie}
      updateValue={updateValue}
      etablissement={etablissement}
   />
</FormInputMesure>
```